### PR TITLE
Add two buttons: one for in-person events and another for both in-person and online events. Display the number of events on the selected date.

### LINKED ISSUE #67: Add two buttons: one for in-person events and another for both in-person and online events
The 'In-person events' button directs to the Map page, and the 'both in-person and online events' button directs to the List View page.

Example below

Image

### PR DESCRIPTION
**Pull request by SachiGoto**
### Feature Description

Two buttons on the home page. One is directed to the map page and another one will be directed to the list view page. 
Screenshot 2023-10-13 at 5 58 55 PM

Also, I implemented a popup message that shows the number of events on the selected date. I thought it will enhance user experience. 

giphy

### Related Issue

close #67 

### Something you want to mention

From 'no events' to 'events' doesn't have smooth transition as you can see in the video. I need to look into it.

### Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/app/map/components/GoogleMaps/index.tsx [hunk 1]
```diff
@@ -148,6 +164,22 @@ export const GoogleMaps = ({ setMapLoaded }: GoogleMapsProps) => {
         </GoogleMap>
       </LoadScript>
 
+      {noEvents && !loading ? (
```
Comment (devjoylee) [inline src/app/map/components/GoogleMaps/index.tsx:167]: the transition issues seems to be caused by the conditional logic for loading state here I guess?.

**Review by devjoylee — APPROVED** [review]
The popup message looks super cool! thank you for enhancing UI by adding them. 👏 
btw, just curious what happened to your branch name. It's very long haha

**Comment by SachiGoto** [comment]
> The popup message looks super cool! thank you for enhancing UI by adding them. 👏 btw, just curious what happened to your branch name. It's very long haha

Ah yeah, I think I auto created the branch name from the related issue. There is a feature to do that right and it was long. I should have changed it to make it shorter haha

File: src/app/map/components/GoogleMaps/index.tsx (hunk 1 above)
Comment (SachiGoto) [reply src/app/map/components/GoogleMaps/index.tsx:167]: Yeah but if I don't add the conditional logic, it loads 'no events' message before 'there is a event' message gets displayed.. But going from 'there is a event'  to 'three is no event'  looks okay. So need to figure out how to render pop up message property. I will play around with it.

**Review by Lada496 — APPROVED** [review]

**Pull request merged**